.gitignore

```diff
@@ -1,4 +1,5 @@
 *.sublime-workspace
+.idea
```

Comment:
If I'm understanding correctly, this is a folder, yeah? As a nit, could this be `.idea/`?